pages/common/autossh.md

```diff
@@ -1,29 +1,29 @@
 # autossh
 
 > Runs, monitors and restarts SSH connections.
-> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all `ssh` flags.
 > More information: <https://harding.motd.ca/autossh>.
```

Comment:
```suggestion
> More information: <https://www.harding.motd.ca/autossh>.
```

Reply:
Wow.  I almost checked the link, but for some reason I forgot.  I'm surprised that the current one doesn't even work!!

Reply:
I was about to replace it completely, when I found the working version during my search. Is this actually the home page of `autossh`? Because if not I'd prefer <https://manned.org/autossh>.